tap_postgres/sync_strategies/incremental.py

```diff
@@ -48,7 +48,7 @@ def sync_table(conn_info, stream, state, desired_columns, md_map):
 
     schema_name = md_map.get(()).get('schema-name')
 
-    escaped_columns = map(partial(post_db.prepare_columns_for_select_sql, md_map=md_map), desired_columns)
+    escaped_columns = list(map(partial(post_db.prepare_columns_for_select_sql, md_map=md_map), desired_columns))
```

Comment:
Curious what's the reason to wrap it with `list()` ?

Reply:
Yeah so basically, `escaped_columns` now is being used in 2 places: 
- The existing original replication sql-build function: `_get_select_sql`
- Our appending sql-build function: `_get_select_latest_sql `

Basically, `map` will return a kind-of iterator object, which will allow us to program in the lazy evaluation way without loading all data into the memory. But the limitation of it is that once we've looped through it, that iterator will be `None` and we can use it twice.
We already loop through the `escaped_columns` in `{','.join(escaped_columns)}` in `_get_select_sql` so when we do it again in `_get_select_latest_sql `, `escaped_columns` will become None, making the SQL wrong.
So that's why I wrap it into the list at the first place to load and persist them within the memory so that we can use it multiple times.